CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# Contributors guide lines.
+
+In un gioco si può lavorare in tante aree diverse: grafica, musica, scrittura, level design e programmazione. Se vuoi lavorare in una o più delle prime 4 aree, il miglior modo per iniziare è
+entrare in contatto con la community tramite i seguenti canali:
+
+- [Gruppo Telegram del progetto](https://t.me/joinchat/Fea7z0xQUHbZn0_wNyj6Vw)
+- [Gruppo Telegram Godot Italia](https://t.me/joinchat/Ga4KigwWn53t2RRfoaPSKg) 
+- [Discord server Godot Italia](https://discord.gg/7YuUqJB)
+
+Una volta dentro puoi subito far parte del team di sviluppo, saperne di più, proporre nuove idee e offrire il tuo lavoro.
+
+Se vuoi programmare o comunque apportare delle modifiche a questa repository, ci sono 2 modi.
+  1. Come prima, entra a far parte della community e del team. Diventerai membro dell'organizzazione su GitHub per avere accesso alla repository. In questo modo potrai aggiungere modifiche alla repository, attraverso branches (modifiche dirette al master sono bloccate in ogni caso).
+  2. Fai una fork di questa repository, aggiungi le tue modifiche e crea una pull request.
+
+Sono benvolute qualsiasi modifiche che aiuti il progetto, anche aggiustare una virgola in questo file
+è una pull request ben accetta. Anche se credi di avere poco da offrire, puoi comunque trovare
+qualcosa da fare con una community molto motivata. 
+
+## Linee guida programmazione
+
+Il gioco è programmato in gdscript e lo stile del codice da seguire (code conventions, naming conventions, whatever conventions) è quello proposto dal team di Godot stesso: [GDScript style guide](https://docs.godotengine.org/en/3.2/getting_started/scripting/gdscript/gdscript_styleguide.html).
+
+Inoltre è richiesta la [tipizzazione](https://docs.godotengine.org/en/3.2/getting_started/scripting/gdscript/gdscript_styleguide.html#static-typing), e i signals vanno utilizzati esclusivamente via
+codice e non editor. Connettendo signals a metodi tramite l'editor rende complicato agli altri
```

Comment:
Non sono d'accordo su questo.l, soprattutto nell' ottica che i level designer magari vorranno assemblare livelli con pezzi esistenti.

Reply:
Puoi essere piu' specifica su cosa non sei d'accordo? Code style scelto, tipizzazione, signals? O tutti e tre?

Reply:
Il commento riguarda il non connettere segnali tramite editor. Io penso che se si danno i pezzi ai LD, si dovrebbe lasciar loro la possibilita' di connettere segnali.

Reply:
Aggiunto un commit per togliere la parte dei segnali. 

Con spoon avevamo pensato di imporre questa cosa e nessuno stava obiettando quindi l'ho aggiunta. Avremmo dovuto prima parlarne con tutti. In ogni caso se verra' aggiunto o no se ne riparla in futuro, quindi per ora levo.